DEVELOPER.md

```diff
@@ -225,7 +225,7 @@ export KUBECONFIG="$(kind get kubeconfig-path --name=kuma)"
 2. Run `Control Plane` on local machine:
 
 ```bash
-make run/k8s
+make kind/deploy/kuma
```

Comment:
this should still be `run/k8s`. This section of instruction is to run Kuma locally and connect to K8S in Kind.